### PR TITLE
feature/mx-1664-use-wikidata-helper

### PR DESCRIPTION
**Pull request by esinsj**
# Changes
extractors now use wikidata helper function

# Fixed
- typo sySnopse

### CODE REVIEW

File: tests/ff_projects/test_extract.py [hunk 1]
```diff
@@ -111,10 +110,10 @@ def test_filter_out_duplicate_source_ids() -> None:
...
-    wikidata_organization: WikidataOrganization,
-) -> None:
+def test_extract_ff_projects_organizations() -> None:
     organizations = extract_ff_projects_organizations(
         [next(extract_ff_projects_sources())]
     )
-    assert organizations["Apple"] == wikidata_organization
+    assert organizations["Apple"] == MergedOrganizationIdentifier(
+        "ga6xh6pgMwgq7DC7r6Wjqg"
+    )
```
Comment (esinsj) [inline tests/ff_projects/test_extract.py:119]: I'm not happy about this solution, but I couldn't find where "wikidata_organization" is defined (in the "mocked_wikidata"?)
Any suggestions?

File: mex/extractors/biospecimen/transform.py [hunk 2]
```diff
@@ -49,7 +49,7 @@ def transform_biospecimen_resource_to_mex_resource(
     person_stable_target_id_by_email = {
         str(p.email[0]): Identifier(p.stableTargetId) for p in mex_persons
     }
-    sysnopse_stable_target_id_by_studien_id = {
+    synopse_stable_target_id_by_studien_id = {
         activity.identifierInPrimarySource: activity.stableTargetId
```
Comment (esinsj) [inline mex/extractors/biospecimen/transform.py:53]: Just a correction of a type (sy*_s_*nopse)

File: mex/extractors/international_projects/extract.py [hunk 3]
```diff
@@ -214,7 +220,11 @@ def get_clean_organizations_names(organizations_str: str) -> list[str]:
         list of clean organizations names
     """
     organizations_str = (
-        organizations_str.replace(",,", ",").replace("»", "").replace("•", "")
```
Comment (esinsj) [inline mex/extractors/international_projects/extract.py:217]: small improvement: add more special characters which can be irgnored. 
(fun fact: "..." find's the rock band "...And You Will Know Us by the Trail of Dead" on wikidata: https://www.wikidata.org/entity/Q426068)

File: tests/grippeweb/test_extract.py [hunk 4]
```diff
@@ -80,72 +81,7 @@ def test_extract_grippeweb_organizations(
...
-            "en": [
-                {"language": "en", "value": "alias_en_1"},
-                {"language": "en", "value": "alias_en_2"},
-                {"language": "en", "value": "alias_en_3"},
-                {"language": "en", "value": "alias_en_4"},
-            ],
-        },
+        "Robert Koch-Institut": MergedOrganizationIdentifier("ga6xh6pgMwgq7DC7r6Wjqg"),
+        "org not in wikidata": MergedOrganizationIdentifier("ga6xh6pgMwgq7DC7r6Wjqg"),
     }
```
Comment (esinsj) [inline tests/grippeweb/test_extract.py:86]: Same as above. Can we use the mocked_wikidata?

File: tests/voxco/test_extract.py [hunk 5]
```diff
@@ -35,11 +35,10 @@ def test_extract_voxco_variables() -> None:
 )
 def test_extract_voxco_organizations(
     voxco_resource_mappings: AnyMappingModel,
-    wikidata_organization: WikidataOrganization,
 ) -> None:
     organizations = extract_voxco_organizations(voxco_resource_mappings)
     assert organizations == {
-        "Robert Koch-Institut": wikidata_organization,
+        "Robert Koch-Institut": MergedOrganizationIdentifier("ga6xh6pgMwgq7DC7r6Wjqg"),
     }
```
Comment (esinsj) [inline tests/voxco/test_extract.py:42]: Same as above. Can we use the mocked_wikidata?

File: tests/ff_projects/test_extract.py (hunk 1 above)
Comment (cutoffthetop) [reply tests/ff_projects/test_extract.py:120]: the ID is built here, deterministic and collision-proof https://github.com/robert-koch-institut/mex-common/blob/0.42.0/mex/common/identity/memory.py#L31-L40
should be fine to hardcode it

**Review by cutoffthetop — CHANGES_REQUESTED** [review]
looks good! i have just a formality: could you update the CHANGELOG and pr-description please?

and one other thing i noticed while reviewing:
these wikidata fixtures here https://github.com/robert-koch-institut/mex-extractors/blob/feature/mx-1664-use-wikidata-helper/tests/conftest.py#L44-L119 seem to be an exact copy of https://github.com/robert-koch-institut/mex-common/blob/0.42.0/mex/common/testing/plugin.py#L125-L193
could you trash those fixtures in mex-extractors (along with the [test_data](https://github.com/robert-koch-institut/mex-extractors/tree/feature/mx-1664-use-wikidata-helper/tests/test_data) folder) and check if the tests still work? in theory they _should_, since the mex-common fixtures are loaded as a plugin anyway.

**Comment by esinsj** [comment]
> looks good! i have just a formality: could you update the CHANGELOG and pr-description please?
> 
> and one other thing i noticed while reviewing: these wikidata fixtures here https://github.com/robert-koch-institut/mex-extractors/blob/feature/mx-1664-use-wikidata-helper/tests/conftest.py#L44-L119 seem to be an exact copy of https://github.com/robert-koch-institut/mex-common/blob/0.42.0/mex/common/testing/plugin.py#L125-L193 could you trash those fixtures in mex-extractors (along with the [test_data](https://github.com/robert-koch-institut/mex-extractors/tree/feature/mx-1664-use-wikidata-helper/tests/test_data) folder) and check if the tests still work? in theory they _should_, since the mex-common fixtures are loaded as a plugin anyway.

I left in the extracted_organization_rki, because that wasn't specified in the mex-common fixtures, but is used by the extractor test.

**Review by cutoffthetop — APPROVED** [review]

**Pull request merged**